File: 3242-count-elements-with-maximum-frequency/count-elements-with-maximum-frequency.py
class Solution(object):
    def maxFrequencyElements(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        frequency = {}
        for num in nums:
            if num in frequency:
                frequency[num] += 1
            else:
                frequency[num] = 1

        max_freq = max(frequency.values())
        total_count = sum(freq for freq in frequency.values() if freq == max_freq)
        return total_count

File: 3242-count-elements-with-maximum-frequency/count-elements-with-maximum-frequency.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var maxFrequencyElements = function(nums) {
    nums.sort((a, b) => a - b);

    let frequency = {};
    let temp_max = 0;

    for (let num of nums) {
        if (Object.keys(frequency).includes(String(num))) {
            frequency[num] += 1;
        } else {
            frequency[num] = 1;
        }
    }

    let max_occ = Math.max(...(Object.values(frequency)));

    for (let occurrence of Object.values(frequency)) {
        if (occurrence === max_occ) {
            temp_max += max_occ;
        }
    }

    return temp_max;
};